docs/api.md

```diff
@@ -0,0 +1,253 @@
+# soundslike - API documentation
+Learn how to use the soundslike API!
+
+First. for the sake of this, im going to use soundslike.com as the url, however localhost:8888 might be the url if you're using docker 
+
+## Section 1. Authentication 
+
+### Getting auth token from api
+
+run 'GET' http://soundslike.com/api/token with a username and password with basic AUTH  
+
+this will return a response of...
+```json
+    {
+        "expiration": 3000,
+        "token": "token_string"
+    }
+``` 
+you'll need to Base64 encode the bearer token provided earlier and include it in all API requests, except user creation 
```

Comment:
I think it makes more sense to explicitly say which API requests require auth instead of saying "all of them except user creation" since we may add more API endpoints later without auth